wiki/Beatmap_Discussion/en.md

```diff
@@ -1,225 +1,183 @@
 ---
-outdated: true
+tags:
+  - Modding v2
+  - mv2
 ---
 
-# Beatmap Discussion
+# Beatmap discussion
 
-Beatmap Discussion (introduced as *Modding v2*) is now live!
+**Beatmap discussion** (also known as *Modding v2*) is a system to manage and simplify the process of [modding](/wiki/Modding). It aims to provide a clear and understandable interface, where the improvement of a beatmap is in focus.
 
-## Submitting a beatmap for Beatmap Discussion
+When a beatmap is [submitted](/wiki/Submission), a discussion page is automatically created alongside the beatmap info page. When the beatmap is updated, the discussion page will be updated along with it. The discussion page can be reached by clicking on the `Discussion` button on the info page. <!--TODO: insert link to beatmap info page article, when that article exists-->
 
-### New beatmaps
+The beatmap discussion page consists of a few elements (from top-to-bottom):
 
-New beatmaps submitted via the Beatmap Submission System will use Beatmap Discussion.
+- Mode selection
+- Beatmap header
+- Difficulty menu
+- Sorting options
+- Visual timeline
+- Beatmap status
+- Modding tabs
+- Submission field
+- Discussions
 
-### Old beatmaps
+You can follow along with our images from [this beatmap](https://osu.ppy.sh/beatmapsets/924551/discussion/) to get familiar with Beatmap Discussions or try opening a discussion page yourself!
```

Comment:
```suggestion
You can follow along with our images from [this beatmap](https://osu.ppy.sh/beatmapsets/924551/discussion/) to get familiar with beatmap discussion or try opening a discussion page yourself!
```

I don't think beatmap discussion needs to be capitalized... maybe I'm wrong?

Reply:
it doesnt